opendevin/core/main.py

```diff
@@ -87,9 +87,13 @@ async def run_agent_controller(
 
     # runtime and tools
     runtime_cls = get_runtime_cls(config.runtime)
-    runtime = runtime_cls(config=config, event_stream=event_stream, sandbox=sandbox)
+    runtime = runtime_cls(
+        config=config,
+        event_stream=event_stream,
+        sandbox=sandbox,
+        plugins=controller.agent.sandbox_plugins,
+    )
     await runtime.ainit()
-    runtime.init_sandbox_plugins(controller.agent.sandbox_plugins)
     runtime.init_runtime_tools(
```

Comment:
why remove plugins but not runtime tools (browser)?

Reply:
We move plugins to be part of `Runtime`'s initialization argument to standardize among Server/EventStream runtime.

Reply:
`runtime_tools` can only be removed after we deprecate `ServerRuntime` unfortunately